### PR TITLE
[DOXIASITETOOLS-291] Add timezone field to site descriptor PublishDat…

### PR DESCRIPTION
**Pull request by michael-o**
…e object and pass onto Velocity tools context

### CODE REVIEW

File: doxia-site-renderer/src/main/java/org/apache/maven/doxia/siterenderer/DefaultSiteRenderer.java [hunk 1]
```diff
@@ -434,6 +435,10 @@ protected Context createToolManagedVelocityContext(SiteRenderingContext siteRend
         Locale locale = siteRenderingContext.getLocale();
         String dateFormat =
                 siteRenderingContext.getDecoration().getPublishDate().getFormat();
+        String timezoneId =
```
Comment (elharo) [inline doxia-site-renderer/src/main/java/org/apache/maven/doxia/siterenderer/DefaultSiteRenderer.java:438]: nit: timeZoneId

File: doxia-site-renderer/src/main/java/org/apache/maven/doxia/siterenderer/DefaultSiteRenderer.java [hunk 2]
```diff
@@ -434,6 +435,10 @@ protected Context createToolManagedVelocityContext(SiteRenderingContext siteRend
...
+                siteRenderingContext.getDecoration().getPublishDate().getTimezone();
+        TimeZone timezone =
```
Comment (elharo) [inline doxia-site-renderer/src/main/java/org/apache/maven/doxia/siterenderer/DefaultSiteRenderer.java:440]: nit: timeZone

**Review by elharo — COMMENTED** [review]
Is there a test for this?

File: doxia-site-renderer/src/main/java/org/apache/maven/doxia/siterenderer/DefaultSiteRenderer.java (hunk 2 above)
Comment (michael-o) [reply doxia-site-renderer/src/main/java/org/apache/maven/doxia/siterenderer/DefaultSiteRenderer.java:440]: I thought about this, but didn't do it in the model: https://maven.apache.org/pom.html `<timezone>`. I can fix Java code, though, if course.

**Comment by michael-o** [comment]
> Is there a test for this?

Yes, but not here. If you run MSITE ITs and change your timezone to far west or far east they will fail. With this change, they will be stable throughout timezones.

**Review by elharo — APPROVED** [review]

**Pull request merged**